srcpkgs/chroot-grep/template

```diff
@@ -1,17 +1,21 @@
 # Template file for 'chroot-grep'
+#
+# THIS PKG MUST BE SYNCHRONIZED WITH "srcpkgs/grep"
+#
 pkgname=chroot-grep
-version=3.3
+version=3.7
 revision=1
 wrksrc="grep-${version}"
 bootstrap=yes
 build_style=gnu-configure
 configure_args="--disable-perl-regexp --disable-nls ac_cv_path_GREP=grep"
-short_desc="The GNU grep utility - for use with xbps-src"
+hostmakedepends="automake"
```

Comment:
I don't think we can have automake as a build dependency in bootstrap packages, if it's only needed for fixing tests, just disable them.

Reply:
Alright.
Should I remove the patches?

Reply:
they fix tests in the grep package too, right? I would leave them there and just remove the symlink

Reply:
It doesn't affect the grep package since it has perl in the checkdepends.

Reply:
then yes, I would just drop them

Reply:
Thanks.